azurerm/resource_arm_container_group_test.go

```diff
@@ -211,6 +211,9 @@ func TestAccAzureRMContainerGroup_linuxComplete(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "container.0.secure_environment_variables.%", "2"),
 					resource.TestCheckResourceAttr(resourceName, "container.0.secure_environment_variables.secureFoo", "secureBar"),
 					resource.TestCheckResourceAttr(resourceName, "container.0.secure_environment_variables.secureFoo1", "secureBar1"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.gpu.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.gpu.0.gpu_count", "1"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.gpu.0.gpu_sku", "K80"),
```

Comment:
given the fields been updated this'll need to be:

```suggestion
					resource.TestCheckResourceAttr(resourceName, "container.0.gpu.0.sku", "K80"),
```